### PR TITLE
(DPS-2941) Placeholder plugin

### PR DESCRIPTION
**Pull request by jakubsikora**

### CODE REVIEW

**Review by dominikslawkowski — APPROVED** [review]
good job!

File: src/components/extensions/Headings.js [hunk 1]
```diff
@@ -127,37 +123,61 @@ export const Warning = styled.div`
...
-//         <Button>Learn more</Button>
-//       </Link>
-//     </SectionColumn>
-//   </SectionBannerWrapper>
-// );
+const SectionBanner = ({ title, desc, to, image }) => (
+  <SectionBannerWrapper>
+    {image && (
+      <SectionColumn style={{ padding: 0 }}>
+        <Link href={to} style={{ display: "block" }}>
```
Comment (arturfracala) [inline src/components/extensions/Headings.js:167]: It's missing `<a>` tag here. Check [documentation](https://nextjs.org/docs/migrating/from-gatsby#links)

File: src/components/extensions/Headings.js [hunk 2]
```diff
@@ -127,37 +123,61 @@ export const Warning = styled.div`
...
+          <BannerImage to={to} />
+        </Link>
+      </SectionColumn>
+    )}
+    <SectionColumn>
+      <h2>{title}</h2>
+      <p>{desc}</p>
+      <Link href={to}>
```
Comment (arturfracala) [inline src/components/extensions/Headings.js:175]: Same thing as above

File: src/hooks/useArticlesInCategory.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ const generatePathsMap = (items, acc = {}) => {
   return acc;
 };
 
-export default (category, currentSlug, currentApiVersion) => {
+const useArticlesInCategory = (category, currentSlug, currentApiVersion) => {
```
Comment (arturfracala) [inline src/hooks/useArticlesInCategory.js:31]: I saw that anonymous functions cause warnings in the console. It's a little bit out of the scope of this PR, but anyway happy to see this :)

**Review by arturfracala — APPROVED** [review]
Really nice work 😄 🎉

**Pull request merged**